apps/portals/arkportal/tsconfig.json

```diff
@@ -22,5 +22,10 @@
     }
   },
   "include": ["src/**/*.tsx", "src/**/*.ts"],
-  "files": ["src/types.d.ts"]
+  "files": ["src/types.d.ts"],
+  "references": [
+    { "path": "../../synapse-portal-framework" },
+    { "path": "../../../packages/synapse-react-client" },
+    { "path": "../../../packages/vite-config" }
+  ]
```

Comment:
Add `references` for incremental type-checking. This does not affect the bug, I only applied this optimization to one portal and forgot to apply it to the rest.